### PR TITLE
feat: [PIE-4903]: pipeline GA changes and fixes

### PR DESCRIPTION
**Pull request by udayvunnam**
### Summary

- pipeline GA changes and fixes

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/70-pipeline/pages/execution-list/ExecutionList.tsx [hunk 1]
```diff
@@ -149,14 +149,12 @@ function ExecutionListInternal(props: ExecutionListProps): React.ReactElement {
 
 export function ExecutionList(props: ExecutionListProps): React.ReactElement {
   return (
-    <>
```
Comment (udayvunnam) [inline src/modules/70-pipeline/pages/execution-list/ExecutionList.tsx:152]: unnecessary fragment

File: src/modules/70-pipeline/pages/pipeline-list/PipelineListPage.tsx [hunk 2]
```diff
@@ -76,7 +76,6 @@ export function PipelineListPage(): React.ReactElement {
   const { showSuccess, showError } = useToaster()
   const { isGitSyncEnabled: isGitSyncEnabledForProject, gitSyncEnabledOnlyForFF } = useAppStore()
   const isGitSyncEnabled = isGitSyncEnabledForProject && !gitSyncEnabledOnlyForFF
-  const [pipelineToDelete, setPipelineToDelete] = useState<PMSPipelineSummaryResponse>()
```
Comment (udayvunnam) [inline src/modules/70-pipeline/pages/pipeline-list/PipelineListPage.tsx:79]: extra state not required, pass this as part of onDeletePipeline

File: src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListCells.tsx [hunk 3]
```diff
@@ -309,7 +307,7 @@ export const MenuCell: CellType = ({ row, column }) => {
           <Menu.Item
             icon="duplicate"
             text={getString('projectCard.clone')}
-            disabled={isGitSyncEnabled || supportingGitSimplification}
+            disabled={isGitSyncEnabled}
```
Comment (udayvunnam) [inline src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListCells.tsx:310]: @sunnykesh-harness FYI

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10711)

**Review by vkbansal-harness — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Pull request merged**